### ISSUE TITLE
obsolete domains on remapping list

### ISSUE THREAD

**Issue by arthur-mozilla** [mozilla-services/shavar-list-creation#82]
Two domains are on the remapping list but not on the blacklist:
* googletagmanager.com
* backtype.com

**Comment by groovecoder** [comment]
Thanks for filing. I have to think if it's worth fixing these on their own, or waiting until we have another list update planned. (Any time we create new lists we cause a big spike of downloads across all Firefox users [and from our servers].)

@englehardt - best I can tell, the `_parse_blocklist` function in `DisconnectParser.py` will simply never try to re-map these domains, now that they are out of the blocklist?

If so, it seems like we can change/fix this, and there will be no effect on the generated list.

**Comment by englehardt** [comment]
@groovecoder that's correct. Removing these two domains should have no impact on the generated lists.

**Comment by boolean5** [comment]
I think this can be closed now, as the `disconnect_mapping.json` file was removed in https://github.com/mozilla-services/shavar-list-creation/pull/121
